test/common.sh

```diff
@@ -80,13 +80,14 @@ function kafka_setup() {
 function install_sources_crds() {
   subheader "Installing Kafka Source CRD"
   kubectl apply -f ${KAFKA_SOURCE_CRD_YAML}
-  # wait_until_pods_running knative-eventing || fail_test "Failed to install the Kafka Source CRD"
-  wait_until_pods_running knative-sources || fail_test "Failed to install the Kafka Source CRD"
+  wait_until_pods_running knative-eventing || fail_test "Failed to install the Kafka Source CRD"
+  # wait_until_pods_running knative-sources || fail_test "Failed to install the Kafka Source CRD"
```

Comment:
Once we get it confirmed by the CI, let remove the deprecated namespace wait to avoid any future confusion.